core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1220,42 +1220,33 @@ public Map<String,String> getTableProperties(final String tableName)
   }
 
   @Override
-  public void setLocalityGroups(String tableName, Map<String,Set<Text>> groups)
+  public void setLocalityGroups(String tableName, Map<String,Set<Text>> groupsToSet)
       throws AccumuloException, AccumuloSecurityException, TableNotFoundException {
     // ensure locality groups do not overlap
-    LocalityGroupUtil.ensureNonOverlappingGroups(groups);
+    LocalityGroupUtil.ensureNonOverlappingGroups(groupsToSet);
 
-    for (Entry<String,Set<Text>> entry : groups.entrySet()) {
-      Set<Text> colFams = entry.getValue();
-      String value = LocalityGroupUtil.encodeColumnFamilies(colFams);
-      setPropertyNoChecks(tableName, Property.TABLE_LOCALITY_GROUP_PREFIX + entry.getKey(), value);
```

Comment:
This is not referencing the `.getKey()` method of the property. Is this a bug? If so, does it exist in earlier versions that need to be fixed (2.1, 3.0)?

Reply:
Looks like its not a bug. `Property.toString()` is as follows:

```java
  @Override
  public String toString() {
    return this.key;
  }
```

Confirmed in an `uno jshell` shell too:
```
jshell> import org.apache.accumulo.core.conf.Property

jshell> String test = Property.TABLE_LOCALITY_GROUP_PREFIX + "";
test ==> "table.group."

jshell> String test2 = Property.TABLE_LOCALITY_GROUP_PREFIX.getKey() + "";
test2 ==> "table.group."
```

Definitely seems better to explicitly use `getKey()` but functionally the same.